src/Sylius/Bundle/OrderBundle/DependencyInjection/SyliusOrderExtension.php

```diff
@@ -36,5 +40,10 @@ public function load(array $config, ContainerBuilder $container): void
 
         $container->setParameter('sylius_order.cart_expiration_period', $config['expiration']['cart']);
         $container->setParameter('sylius_order.order_expiration_period', $config['expiration']['order']);
+
+        $container->registerForAutoconfiguration(CartContextInterface::class)
```

Comment:
Just my personal view, but maybe it would be better to order chained functions like this:
```php
$container
    ->registerForAutoconfiguration(CartContextInterface::class)
    ->addTag(RegisterProcessorsPass::CART_CONTEXT_SERVICE_TAG)
;
```

WDYT?

Reply:
done (same for `SyliusFixtureBundle` for consistency)